html-forms/index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Forms</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+    <form>
+      <div>
+        <label>
+          Email
+        </label>
+      </div>
+      <input type="email" name="email">
+      <div>
+        <label for="coding is cool">Why is coding cool?</label>
```

Comment:
There should not be spaces in the value here, you should use `kebab-case`. Spaces in attribute values generally means multiple values are being assigned to that attribute.

Reply:
Also, it is not configured correctly, just like the previous `label`